projection.tex

```diff
@@ -113,7 +113,7 @@ \subsection{Vector form}
 \nabla \cdot \mathbf{v}=0 \label{e:continuity_vec}
 \end{align}
 %
-\section{Projection algorithm}
+\section{Projection algorithm \cite{ferziger_cfd_2019}}
 Velocity at $n^{th}$ time-step is denoted as $\mathbf{v}^n$. Convection term is 
```

Comment:
Please write something about the numerical method you are employing in this note.  Conventionally we (scientists and engineers) call them numerical methods, not algorithms.  The term algorithm is mostly used by programmers.